README.md

```diff
@@ -55,7 +55,7 @@ manager:
 brew tap charmbracelet/tap && brew install charmbracelet/tap/soft-serve
 
 ## Windows (with Winget)
-winget install charmbracelet.soft-serve
```

Comment:
Hey, I know it's been a while. Thank you for this PR, I was just wondering if there's any particular reason why you removed the publisher portion of the package name in this commit? I know nothing about winget, so please don't take this as anything other than me trying to understand how the naming works 😂

Reply:
Hi, Winget supports installing applications via the full package identifier or a unique moniker (which is defined in the locale file). Moniker is a unique name that can be used by an application. So instead of specifying the full identifier `winget install charmbracelet.soft-serve` you can use `winget install soft-serve` to install it with ease instead.

i.e. https://github.com/microsoft/winget-pkgs/blob/4d0fcf398afe265216966905f7e6aa3c0a6e3a41/manifests/c/charmbracelet/soft-serve/0.7.3/charmbracelet.soft-serve.locale.en-US.yaml#L12

In case there is another application with the same name/moniker in future we could switch this back to the full package identifier, currently this would work fine.